modell/src/test/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/VilkårsvurderderingstegTest.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.tilbakekreving.behandling.saksbehandling
+
+import io.kotest.matchers.shouldBe
+import no.nav.tilbakekreving.HistorikkStub
+import no.nav.tilbakekreving.februar
+import no.nav.tilbakekreving.januar
+import no.nav.tilbakekreving.kravgrunnlag
+import no.nav.tilbakekreving.kravgrunnlagPeriode
+import no.nav.tilbakekreving.til
+import org.junit.jupiter.api.Test
+
+class VilkårsvurderderingstegTest {
```

Comment:
Skrivefeil på resten av filen også
```suggestion
class VilkårsvurderingStegTest {
```